crates/solvers-dto/src/auction.rs

```diff
@@ -34,8 +34,6 @@ pub struct Order {
     pub sell_amount: U256,
     #[serde_as(as = "HexOrDecimalU256")]
     pub buy_amount: U256,
-    #[serde_as(as = "HexOrDecimalU256")]
-    pub fee_amount: U256,
```

Comment:
How confident are we that this breaking change is not going to shoot us in the foot? I think it would be ok to leave this value on the API level but simply always set it to 0 in code (and document that it's deprecated and always 0 in the API spec). For our internal models (domain, etc), we would want to remove this field however.

The frontend team would like to re-vamp our API and I think it would be ok to defer breaking changes thereto.

Reply:
The mentioned field is part of the solvers API. Not sure what frontend has with it? If you got confused and referred to the `model` data struct than yes, we still want to keep `fee_amount` there.
Solvers API should be kept clean, this moment is good enough to make this change as any other IMO (actually, would have been better if we made this change before solvers moved to this API 😄  buy hey...)

Reply:
Sorry, wrong file (I wanted to comment on the one above: crates/orderbook/src/dto/order.rs), there the field is `user_fee`. Just want to make sure only our frontend depends on this field and can live without it.

Reply:
This is a similar change we had few weeks ago with `solver_fee`: https://github.com/cowprotocol/services/pull/2272 No major problems arose except shadow competition was down and alerter being down AFAIR.

Reply:
This time I would expect only shadow to be down because alerter is not deserializing `user_fee`.